Makefile

```diff
@@ -610,7 +569,7 @@ deb: test soter_static themis_static soter_shared themis_shared collect_headers
 	@find $(BIN_PATH) -name \*.deb
 
 
-rpm: test themis_static themis_shared soter_static soter_shared collect_headers install_shell_scripts strip symlink_realname_to_soname
+rpm: prepare_tests_basic themis_static themis_shared soter_static soter_shared collect_headers install_shell_scripts strip symlink_realname_to_soname
```

Comment:
prepare_tests_baisc -> test_basic. run tests before building package

Reply:
I can do it, but this PR won't be 'refactoring' anymore (because we're changing logic here, not just renaming)

Reply:
logic the same - test code before package building but before this logic was broken, but now you fix it and it's great

to finish this refactoring we should use new `make test` in our `circle.yml` instead of current commands:
```
...
    - build/tests/soter_test
    - build/tests/themis_test
    - build/tests/themispp_test
    - build/tests/pythemis_test.sh
    - build/tests/phpthemis_test.sh
    - build/tests/node.sh
...
```
what you think?